docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,49 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- The `cy.log()`command will now display a line break where the `\n` character
+  is used. Addressed in
+  [#25467](https://github.com/cypress-io/cypress/pull/25467).
+- `.jsx`/`.tsx` file extensions will be utilized when generating a new empty
+  spec file inside component testing if the project contains at least one file
+  with those extensions. Addressed in
+  [#25318](https://github.com/cypress-io/cypress/pull/25318).
+- Added new `experimentalSkipDomainInjection` flag to disable Cypress from
+  setting `document.domain` on injection. Addressed in
+  [#25307](https://github.com/cypress-io/cypress/pull/25307).
+- Added new `experimentalMemoryManagement` flag to improve memory management in
+  Chromium-based browsers. Addressed in
+  [#25462](https://github.com/cypress-io/cypress/pull/25462).
+- Video output link in `cypress run` mode has been added to it's own line to
+  make the video output link more easily clickable in the terminal. Addressed in
+  [#23915](https://github.com/cypress-io/cypress/pull/23915).
+- Support `data-qa` selector in Selector Playground in addition to `data-cy`,
+  `data-test` and `data-testid`. Addressed in
+  [#25475](https://github.com/cypress-io/cypress/pull/25475).
+  - `.as()` now accepts an options argument, allowing test writers to store an
+    alias as a "query" or "static". Addressed in
+    [#25251](https://github.com/cypress-io/cypress/pull/25251).
+
+**Bugfixes:**
+
+- Fixes an issue where component tests could incorrectly treat new major
+  versions of certain dependencies as supported. Addressed in
+  [#25380](https://github.com/cypress-io/cypress/pull/25380).
+- Fixes an issue where spaces on new lines weren't kept in the test runner.
+  Addressed in [#24687](https://github.com/cypress-io/cypress/pull/24687).
+- Fixes an issue where Angular component testing projects would fail to
+  initialize if an unsupported browserslist entry was specified in the project
+  configuration. Addressed in
+  [#25500](https://github.com/cypress-io/cypress/pull/25500).
+
+**Misc**
+
+- Documentation for `createOutputSpy` inside Angular component testing has been
+  corrected to reference the property inside `componentProperties`. Addressed in
+  [#25359](https://github.com/cypress-io/cypress/pull/25359)
+- Vite 4+ will no longer show a warning when used for component testing
```

Comment:
This sounds like a feature for adding official support for vite 4.

Reply:
Yes I would phrase it as `Official support for Vite 4 projects for component testing`. I would drop the `+` as we don't support anything past v4.